models/courier/Courier.model.js

```diff
@@ -41,9 +43,20 @@ const CourierSchema = new Schema(
       default: false,
     },
   },
-  { versionKey: false, timestamps: true }
+  {
+    versionKey: false,
+    timestamps: true,
+    id: true,
```

Comment:
Just realized that we have a piece of code that repeats in every Mongoose model - the part that changes the ID from underscored to non-underscored. We can extract this into a constant and import it into our models. We can do this later if preferred.

Reply:
Everyone merges models, and one of us does that task.